### PR TITLE
Feature/WLZAA-96 add cookies policy

### PR DESCRIPTION
**Pull request by susomejias**
# Don't forget to add Jira issue related code to the title 😇

## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix

## Description

- Add cookies policy

### Mobile

image

image

image

### Tablet

image

image

image

### Desktop

image

image

image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -124,3 +124,4 @@ out
 build
 __tests__/e2e/screenshots
 __tests__/e2e/videos
+yarn.lock
```
Comment (Miangame) [inline .gitignore:127]: El yarn lock si se deberia subir

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -51,6 +58,7 @@ const ExtendedApp = (props: AppProps) => {
   }
 
   useEffect(() => {
+    console.warn(getCookieConsentValue());
```
Comment (Miangame) [inline src/pages/_app.tsx:61]: 🔥

**Review by Miangame — CHANGES_REQUESTED** [review]

File: src/pages/_app.tsx (hunk 2 above)
Comment (susomejias) [reply src/pages/_app.tsx:61]: uppss se coló!!

File: .gitignore (hunk 1 above)
Comment (susomejias) [reply .gitignore:127]: Opss cierto

**Review by Miangame — APPROVED** [review]

**Pull request merged**